dandiapi/api/management/commands/migrate_schema_versions.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import logging
+
+from dandischema.consts import DANDI_SCHEMA_VERSION
+from django.db import transaction
+import djclick as click
+
+from dandiapi.api.models import Version
+from dandiapi.api.services import audit
+from dandiapi.api.services.metadata import validate_version_metadata
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument(
+    'dandisets',
+    type=click.INT,
+    nargs=-1,
+)
+@click.option(
+    '-a',
+    '--all',
+    'include_all',
+    is_flag=True,
+    help='Run on all dandisets.',
+)
+def migrate_schema_versions(dandisets: tuple[int, ...], *, include_all: bool):
+    if bool(dandisets) == include_all:
+        raise click.ClickException("Must specify exactly one of 'dandisets' or --all")
+
+    versions = Version.objects.filter(version='draft')
+    if dandisets:
+        versions = versions.filter(dandiset_id__in=dandisets)
+
+    logger.info('Running migration for %s versions', versions.count())
+
+    for version in versions.iterator():
+        logger.info('-----------------------------------------')
+        with transaction.atomic():
+            locked_version = Version.objects.select_for_update().get(id=version.id)
+            locked_version.metadata['schemaVersion'] = DANDI_SCHEMA_VERSION
```

Comment:
I do not see here an actual [`dandischema.metadata.migrate`](https://github.com/dandi/dandi-schema/blob/master/dandischema/metadata.py#L340) call which would migrate the model... we cannot just slap a new version onto an older one - some fields might need to change.